src/js/components/AirPlayButton.js

```diff
@@ -89,12 +89,17 @@ AirPlayButton = {
          return;
       }
 
-      mediaEl.addEventListener('webkitplaybacktargetavailabilitychanged', function(event) {
+      function _onTargetAvailabilityChanged(event) {
```

Comment:
Just a minor nit: we don't normally start variable/function/method names with an underscore unless they are private or protected.